gradle/publish.gradle

```diff
@@ -0,0 +1,26 @@
+// run `./gradlew publishToMavenLocal` to install artifact to your Local Maven Repository
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            pom {
+                groupId = 'org.jspecify.experimentaldonotuse'
+                name = 'Code Analysis Annotations'
+                description = 'An artifact of well-named and well-specified annotations to power static analysis checks'
+                url = 'https://jspecify.org/'
```

Comment:
We have made jspecify.org redirect to jspecify.dev. That means that both are fine, but maybe it's nice to link directly to the .dev. domain?